### PR TITLE
[DO NOT MERGE] testing #445

### PR DESCRIPTION
**Pull request by michelley-an**
## Overview

Testing the changes in #445 to augment the CI, along with syntax fixes.

## Update type - breaking / non-breaking

- [x] Minor bug fix
- [ ] Documentation improvements
- [ ] Quality of Life improvements
- [ ] New features (non-breaking change)
- [ ] New features (breaking change)
- [ ] Other (non-breaking change)
- [ ] Other (breaking change)
- [ ] Release preparation

## What does this solve?

PR #445 

## What databases have you tested with?

- [ ] Snowflake
- [ ] Google BigQuery
- [ ] Databricks
- [ ] Spark
- [ ] N/A

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: "dbt_artifacts"
-version: "2.6.4"
+version: "2.7.1"
 config-version: 2
 require-dbt-version: [">=1.3.0", "<1.9.0"]
```
Comment (michelley-an) [inline dbt_project.yml]:
```suggestion
require-dbt-version: [">=1.3.0", "<=1.9.0"]
```

File: .github/workflows/ci_test_package.yml [hunk 2]
```diff
@@ -113,7 +113,7 @@ jobs:
     strategy:
       fail-fast: false # Don't fail one DWH if the others fail
       matrix:
-        warehouse: ["snowflake", "bigquery", "postgres", "sqlserver"]
+        warehouse: ["snowflake", "bigquery", "postgres"]
         # When supporting a new version, update the list here
         version: ["1_3_0", "1_4_0", "1_5_0", "1_6_0", "1_7_0", "1_8_0"]
```
Comment (michelley-an) [inline .github/workflows/ci_test_package.yml]:
```suggestion
        version: ["1_3_0", "1_4_0", "1_5_0", "1_6_0", "1_7_0", "1_8_0", "1_9_0"]
```

File: dbt_project.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: "dbt_artifacts"
-version: "2.6.4"
+version: "2.7.1"
 config-version: 2
-require-dbt-version: [">=1.3.0", "<1.9.0"]
+require-dbt-version: [">=1.3.0", "<=1.9.0"]
```
Comment (michelley-an) [inline dbt_project.yml]:
```suggestion
require-dbt-version: [">=1.3.0", "<1.10.0"]
```

**Comment by KijkEr** [comment]
Try this:
image

Add dbt-core~=1.8.0 to the deps of the test in tox.ini.
I've already added this change in #445

**Pull request closed without merging**